src/accessors/java/org/spongepowered/common/accessor/entity/player/ServerPlayerEntityAccessor.java

```diff
@@ -22,33 +22,15 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package org.spongepowered.common.mixin.api.mcp.item;
+package org.spongepowered.common.accessor.entity.player;
 
-import net.minecraft.entity.merchant.IMerchant;
-import net.minecraft.item.MerchantOffer;
-import net.minecraft.item.MerchantOffers;
-import org.spongepowered.api.item.merchant.Merchant;
-import org.spongepowered.api.item.merchant.TradeOffer;
-import org.spongepowered.api.item.merchant.TradeOfferListMutator;
+import net.minecraft.entity.player.ServerPlayerEntity;
+import net.minecraft.util.math.Vec3d;
 import org.spongepowered.asm.mixin.Mixin;
-import org.spongepowered.common.item.util.TradeOfferUtil;
-import java.util.List;
-import java.util.Random;
+import org.spongepowered.asm.mixin.gen.Accessor;
 
-@Mixin(MerchantOffers.class)
-public interface MerchantOffersMixin_API extends TradeOfferListMutator {
-
-    @Override
-    default void accept(final Merchant owner, final List<TradeOffer> tradeOffers, final Random random) {
-        final MerchantOffers tempList = new MerchantOffers();
-        for (final TradeOffer offer : tradeOffers) {
-            tempList.add(TradeOfferUtil.toNative(offer));
-        }
-        addMerchantRecipe((IMerchant) owner, tempList, random);
-        tradeOffers.clear();
-        for (final MerchantOffer recipe : tempList) {
-            tradeOffers.add(TradeOfferUtil.fromNative(recipe));
-        }
-    }
+@Mixin(ServerPlayerEntity.class)
+public interface ServerPlayerEntityAccessor {
```

Comment:
Do we not even have one of these from before?

Reply:
I dont see another accessor for ServerPlayerEntity